### ISSUE TITLE
Port: Improve player movement

### ISSUE THREAD

**Issue by ChazyTheBest** [LightsHope/server#927]
I saw the videos from the old Nostalrius team where they fixed some path finding bugs and I thought this could be also interesting to implement, but in this case it'd be a port since it's already in the works by a TC member: https://github.com/TrinityCore/TrinityCore/pull/18771

The idea is to fix the player movement system so it's smoother and synchronize all clients with server (currently they're not).

I'm just suggesting this and see if you guys would like to see it in Lights Hope.

**Comment by brotalnia** [comment]
It's a giant pull request for another core, that targets a different expansion. Not an easy thing to port.

**Comment by ChazyTheBest** [comment]
Yes, I'm aware of that. But with a bit of luck, CMangos port it themselves and then it's just easier. Also, let's see what others say (checkout the videos).
